Index: mg/fileio.c
===================================================================
--- mg.orig/fileio.c
+++ mg/fileio.c
@@ -268,6 +268,13 @@ fbackupfile(const char *fn)
 	}
 	serrno = errno;
 	(void) fchmod(to, (sb.st_mode & 0777));
+	/* copy the mtime to the backupfile */
+	{
+		struct timespec new_times[2];
+		new_times[0] = sb.st_atim;
+		new_times[1] = sb.st_mtim;
+		futimens(to, new_times);
+	}
 	close(from);
 	close(to);
 	if (nread == -1) {
